docs/source/mocks.rst

```diff
@@ -133,4 +133,24 @@ Setting the value of multiple variables
       [myKey]: 1234
     }
   })
-  
\ No newline at end of file
+  
+Getting the value of an internal variable
+********************
+
+.. code-block:: typescript
+
+  const myUint256 = await myMock.getVariable('myUint256VariableName');
+
+Getting the value of an internal mapping given the value's key
+********************
+
+.. code-block:: typescript
+
+  const myMappingValue = await myMock.getVariable('myMappingVariableName', [mappingKey]);
+
+Getting the value of an internal nested mapping given the value's keys
+********************
+
+.. code-block:: typescript
+
+  const myMappingValue = await myMock.getVariable('myMappingVariableName', [mappingKeyA, mappingKeyB]);
```

Comment:
let's please add a warning, mentioning the things that don't work. Will links to PRs